templates/docs/examples/patterns/navigation/sliding.html

```diff
@@ -0,0 +1,118 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Navigation / Sliding{% endblock %}
+
+{% block standalone_css %}patterns_navigation{% endblock %}
+
+{% block content %}
+<header id="navigation" class="p-navigation is-dark p-navigation--sliding">
```

Comment:
In Vanilla we don't duplicate component and component+modifier class names, so it would need to be `p-navigation--sliding` only.

But to make it work we need to make sure `p-navigation--sliding` implements all the same styles as `p-navigation` does, so it likely means some additional selectors.

Reply:
And we don't want to use the `&--sliding {` syntax, right?

Reply:
Yes, but we are not using that part of BEM apparently 🤷 , it's been like that from beginning of Vanilla. We should not change it now for one component.

So, when you have `p-navigation--sliding` it should not need `p-navigation` class, it should already implement everything `p-navigation` needs. Either by SASS extend, or by `p-navigation, p-navigation--sliding` kind of selectors.

I agree, that it would probably be better to strictly follow BEM in this regard, but it's not a point to change it now.

Reply:
How is `.is-dark` different then? Shouldn't it be `p-navigation--dark`?